README.md

```diff
@@ -72,34 +74,44 @@ echo "your keystore password" > `pwd`/password.txt
 
 Then run the faucet application without the wallet command-line flags:
 ```bash
-./eth-faucet -httpport 8080
+./lsk-faucet -httpport 8080
 ```
 
 **Optional Flags**
 
 The following are the available command-line flags(excluding above wallet flags):
 
 | Flag              | Description                                      | Default Value |
-|-------------------|--------------------------------------------------|---------------|
+| ----------------- | ------------------------------------------------ | ------------- |
 | -httpport         | Listener port to serve HTTP connection           | 8080          |
 | -proxycount       | Count of reverse proxies in front of the server  | 0             |
-| -faucet.amount    | Number of Ethers to transfer per user request    | 1             |
+| -token-address    | Token contract address                           |               |
```

Comment:
We should already specify the contract address here? However, I'm unsure if we should specify the L1/L2/both contracts. Please double check.
Please refer [this](https://github.com/LiskHQ/lisk-contracts/blob/main/deployment/addresses/sepolia/addresses.md) documentation.
```suggestion
| -token-address    | Token contract address                           |               |
```